packages/marathon/build

```diff
@@ -19,6 +19,8 @@ StartLimitInterval=0
 RestartSec=15
 PermissionsStartOnly=True
 EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=-/opt/mesosphere/etc/marathon
```

Comment:
Note: I'm not putting any contents in `/opt/mesosphere/etc/marathon` here, but it is prefaced with a `-` so it won't error if the file doesn't exist.